buttercup/strings/en_US.yaml

```diff
@@ -26,4 +18,4 @@ restrictor:
   wrong_prefix: |
     Hey <@{0}>, it appears that you have attempted to change your nickname, but forgot the initial `/` in your nickname. Make sure that your nickname starts with `/u/` and not `u/`!
   new_lifted: |
-    Welcome <@{0}> to the Transcribers of Reddit Discord! Since you have accepted our Code of Conduct and changed your nickname to the correct format, you now have unrestricted access to our Discord. Thanks for your time and enjoy your time in here!
\ No newline at end of file
+    Welcome <@{0}> to the Transcribers of Reddit Discord! Since you have changed your nickname to the correct format, you now have unrestricted access to our Discord. Thanks for your time and enjoy your time in here!
```

Comment:
This is missing a new line at the end of the file

Reply:
I have added this newline!

Reply:
Thanks, looks good now!